.github/workflows/flutter.yml

```diff
@@ -75,10 +75,10 @@ jobs:
         env:
           TOTAL: ${{ steps.analysis.outputs.total }}
           TOTAL_MAX: ${{ steps.analysis.outputs.total_max }}
-        # TODO: Once 1.0.0 lands, change to 100 
+        # TODO: Once 4.0.0 lands, change to 100 
```

Comment:
This comment was here because versions 0.0.x got discounted points.
We can already version it 4.0.0 since it'll be push out with the same version as `sentry`

```suggestion
```